tests/e2e/special-pages.spec.js

```diff
@@ -3,51 +3,56 @@ import {test, expect} from './tools/lib/test-utils.js';
 test.useAdminLoggedIn();
 
 test('Test special pages (Act and Explore)', async ({page, requestUtils, admin}) => {
-  // Create 2 new pages.
-  const actPage = await requestUtils.createPage({
-    title: 'Act page test',
-    content: '<!-- wp:paragraph --><p>Random content</p><!-- /wp:paragraph -->',
-    status: 'publish',
-  });
-
-  const explorePage = await requestUtils.createPage({
-    title: 'Explore page test',
-    content: '<!-- wp:paragraph --><p>Random content</p><!-- /wp:paragraph -->',
-    status: 'publish',
-  });
+  // Check if new IA is enabled, in which case the Act and Explore pages have been removed.
   await admin.visitAdminPage('admin.php', 'page=planet4_settings_navigation');
-  await page.waitForSelector('#act_page');
-  await page.waitForSelector('#explore_page');
```

Comment:
you need a `await page.reload();` here so that your newly posted pages appear in the select options

Reply:
Updated!